apps/ui/src/composables/useSpaceAlerts.ts

```diff
@@ -0,0 +1,70 @@
+import { SPACE_ALERTS } from '@/helpers/constants';
+import { offchainNetworks } from '@/networks';
+import { Space } from '@/types';
+
+const OVERRIDING_STRATEGIES: string[] = [
+  'aura-vlaura-vebal-with-overrides',
+  'balance-of-with-linear-vesting-power',
+  'balancer-delegation',
+  'cyberkongz',
+  'cyberkongz-v2',
+  'delegation',
+  'delegation-with-cap',
+  'delegation-with-overrides',
+  'erc20-balance-of-delegation',
+  'erc20-balance-of-fixed-total',
+  'erc20-balance-of-quadratic-delegation',
+  'erc20-votes-with-override',
+  'esd-delegation',
+  'ocean-dao-brightid',
+  'orbs-network-delegation',
+  'api-v2-override',
+  'rocketpool-node-operator-delegate-v8',
+  'eden-online-override'
```

Comment:
This is missing `split-delegation`
```suggestion
  'eden-online-override',
  'split-delegation'
```